docs/schema-loader.md

```diff
@@ -387,9 +386,10 @@ $ java -jar scalardb-schema-loader-<VERSION>.jar --jdbc -j <JDBC_URL> -u <USER>
 
 <div class="notice--info">{{ notice--info | markdownify }}</div>
 
-### Repair tables
+### Repair namespaces and tables
 
-You can repair the table metadata of existing tables by using the properties file. To repair table metadata of existing tables, run the following command, replacing the contents in the angle brackets as described:
+You can repair namespaces and tables by using the properties file. Namespaces and tables can be in an unknown state such as namespace or table exists in the underlying storage but not their ScalarDB metadata or vice versa. This command will recreate the namespaces, the tables, the secondary indexes and their metadata if necessary.
+To repair them, run the following command, replacing the contents in the angle brackets as described:
```

Comment:
I revised this part for flow. I don't think I changed the meaning, but please confirm that it's still accurate🙇🏻‍♂️

```suggestion
You can repair namespaces and tables by using the properties file. The reason for repairing namespaces and tables is because they can be in an unknown state, such as a namespace or table exists in the underlying storage but not its ScalarDB metadata or vice versa. 

Repairing the namespaces, the tables, the secondary indexes, and their metadata requires re-creating them. To repair them, run the following command, replacing the contents in the angle brackets as described:
```

Reply:
Thanks for all your comments.
In the first sentence of the second paragraph of your suggestion, I think we need `if necessary` otherwise, it implies things will be recreated every time even though it is not. What do you think ?

```suggestion
You can repair namespaces and tables by using the properties file. The reason for repairing namespaces and tables is because they can be in an unknown state, such as a namespace or table exists in the underlying storage but not its ScalarDB metadata or vice versa. 
Repairing the namespaces, the tables, the secondary indexes, and their metadata requires re-creating them if necessary. To repair them, run the following command, replacing the contents in the angle brackets as described:
```

Reply:
Ok👍, thank you. 
I applied all your suggestions. https://github.com/scalar-labs/scalardb/pull/1300/commits/858fab44e062adfc85c79835612e9431c99b1bf9